### PR TITLE
CI: Publish to bluesky.github.io. Always build.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -117,18 +119,17 @@ script:
   - coverage report
   # Make sure only the primary build uploads to codecov
   - |
-    if [ $BUILD_DOCS ]; then
+    if [ $PUBLISH_DOCS ]; then
```
Comment (mrakitin) [inline .travis.yml]: Should not it be something like
```suggestion
    if [ ! -z "$PUBLISH_DOCS" ]; then
```
?

Comment (danielballan) [reply .travis.yml]: As written the docs are published if `PUBLISH_DOCS` is _set_, so we turn it off by simply not setting that variable. But I agree that this is clearer.

File: .travis.yml [hunk 2]
```diff
@@ -117,18 +119,17 @@ script:
...
+  - pip install -r docs-requirements.txt
+  - make -C doc html
   - |
-    if [ $BUILD_DOCS ]; then
-      pip install -r docs-requirements.txt
-      pushd doc
-      make html
-      popd
-      # Publish docs.
+    if [ $PUBLISH_DOCS ]; then
```
Comment (mrakitin) [inline .travis.yml]:
```suggestion
    if [ ! -z "$PUBLISH_DOCS" ]; then
```

File: .travis.yml (hunk 1 above)
Comment (mrakitin) [reply .travis.yml]: Indeed, they work identically. Tested it with this script and changing the `VAR`:
```bash
#!/bin/bash

VAR=test

echo "Old approach"
if [ $VAR ]; then
    echo "VAR set to $VAR"
else
    echo "VAR is empty"
fi

echo "New approach"
if [ ! -z "$VAR" ]; then
    echo "VAR set to $VAR"
else
    echo "VAR is empty"
fi
```
Outputs:
```bash
$ bash test.sh
Old approach
VAR is empty
New approach
VAR is empty
# Update the 'VAR' to be 'test'
$ bash test.sh
Old approach
VAR set to test
New approach
VAR set to test
```

**Pull request merged**